File: templates/griffon-javafx-ios-templates/templates/subtmpl-artifact/IntegrationTest.java
package ${project_package};

import griffon.javafx.test.GriffonTestFXRule;
import org.junit.Rule;
import org.junit.Test;

import static org.junit.Assert.fail;

public class ${project_class_name}IntegrationTest {
    @Rule
    public GriffonTestFXRule testfx = new GriffonTestFXRule("mainWindow");

    @Test
    public void smokeTest(){
        fail("Not yet implemented!");
    }
}

File: templates/griffon-javafx-kotlin-templates/templates/subtmpl-artifact/IntegrationTest.kt
package ${project_package}

import griffon.javafx.test.GriffonTestFXRule
import org.junit.Rule
import org.junit.Test

import org.junit.Assert.fail

class ${project_class_name}IntegrationTest {
    @Rule @JvmField
    val testfx: GriffonTestFXRule = GriffonTestFXRule("mainWindow")

    @Test
    fun smokeTest() {
        Assert.fail("Not yet implemented!")
    }
}
